pages/docs/user-docs/action-build.md

```diff
@@ -1,303 +1,185 @@
 ---
-title: Build an Image
+title: Build a Container
 sidebar: user_docs
 permalink: docs-build
 folder: docs
+toc: false
 ---
 
-Build is the process where we install an operating system and then configure it appropriately for a specified need. To do this we use a [Singularity recipe](/docs-recipes) file (a text file called `Singularity`) which is a recipe of how to specifically build the container. Here we will overview the usage, best practices, and quick examples.
+The `build` command lets you obtain a container. 
 
```

Comment:
I would just delete "The build command lets you obtain  container." Obtain is more similar to pull, and building is what it is... building. It's a cleaner, strong start to just begin with "Build" is the "Swiss..."